.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+ - [ ] Ready
+ - [ ] In development
+ - [ ] Hold
```

Comment:
```suggestion
```

I don't think we need this since setting the PR as a draft already replaces the last two options. And the PR is already in ready state if it's waiting for approval.

Reply:
Makes Sense

Reply:
I'll commit the changes while I'm here.